docs/mergeop.md

```diff
@@ -0,0 +1,301 @@
+# Requirements
+1. "Fast" merges that don't duplicate data on disk when using the right underlying snapshotter and have an execution time less than or equal to the previous llb.Copy implementation in vast majority of cases.
```

Comment:
The perf requirement is that `llb.Copy(src, dest)` has same complexity as `Merge(src, llb.Copy(src, scratch))`

Reply:
Updated with that